rclcpp/include/rclcpp/service.hpp

```diff
@@ -311,7 +313,7 @@ class Service
   : ServiceBase(node_handle), any_callback_(any_callback)
   {
     using rosidl_typesupport_cpp::get_service_type_support_handle;
-    auto service_type_support_handle = get_service_type_support_handle<ServiceT>();
+    service_type_support_handle_ = get_service_type_support_handle<ServiceT>();
```

Comment:
Same here

Reply:
Fixed in 01aa7e0.  Actually, I also noticed another problem here which was that we weren't setting up the handle for all Service class constructors.  I also fixed that.